helm/aws-operator-chart/templates/deployment.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   namespace: {{ .Values.namespace }}
   labels:
     app: aws-operator
+    version: {{ .Chart.Version }}
 spec:
```

Comment:
Don't we have selector in the deployment?

Reply:
There are no selector in the deployment, this is just to align deployment and pod labels.

Reply:
Do you know how it works? Does it take its own labels and create selector out of that by default?

Reply:
When deployment selector is empty, it takes the pod labels by default. https://godoc.org/k8s.io/api/extensions/v1beta1#ReplicaSetSpec

Selector must be specified, starting from k8s 1.15 in api version `apps/v1beta2`, but we are still using `extensions/v1beta1` here. https://gigantic.slack.com/archives/C12SG2929/p1574337314118300

All good, let's move on!

Reply:
👍